pipeline.py

```diff
@@ -89,14 +89,12 @@ def get_canonical_isbn(genome):
     if genome['pagetypes']['modules']['backpage_isbn']['results']:
         b_isbns = genome['pagetypes']['modules']['backpage_isbn']['results']
 
-    if c_isbns and b_isbns:
+    if c_isbns and b_isbns and any(x in c_isbns for x in b_isbns):
         return [x for x in c_isbns if x in b_isbns][0]
-    elif b_isbns:
-        return b_isbns[-1]
     elif c_isbns:
         return c_isbns[0]
-    else:
-        return None
+    elif b_isbns:
```

Comment:
we may want some comments to describe what's going on here -- if I had to ask, others may have the same question :)

copyright v. back page strategy